tests/post/features/authentication.feature

```diff
@@ -11,6 +11,7 @@ Feature: Authentication is up and running
 
     Scenario: Reach the OpenID Config
         Given the Kubernetes API is available
+        And the control-plane Ingress container is Ready
```

Comment:
Shouldn't we add this line in each and every scenarios below ?
Maybe, we can even add a `Background` add the beginning of the feature, since we always need the same things:
```
    Background:
        Given the Kubernetes API is available
        And the control-plane Ingress container is Ready
        And the control-plane Ingress path '/oidc' is available
        And pods with label 'app.kubernetes.io/name=dex' are 'Ready'

    Scenario: Reach the OpenID Config
        Then we can reach the OIDC openID configuration
```

Reply:
Agreed that we could improve the entire tests suite and not limited to authentication by adding `Background`  scenarios.

Worth creating an issue to track this refactoring.
I have added the line to each scenario for now

Reply:
So instead of adding the `Given` to each line, I merged the check for container status in the same function that checks for pod status using submethods

Reply:
Let's create this tracking issue then @Ebaneck